src/keygen/participant.rs

```diff
@@ -122,7 +122,6 @@ impl ProtocolParticipant for KeygenParticipant {
             MessageType::Keygen(KeygenMessageType::R3Proof) => {
                 self.handle_round_three_msg(rng, message, input)
             }
-            MessageType::Keygen(_) => Err(InternalError::MessageMustBeBroadcasted)?,
```

Comment:
This appears to have been leftover code from before message types for broadcast changed

Reply:
Oh yeah, I made #266 because I thought there might be some extra conditions in here. Can you mark it as being closed by this PR?